src/homedelivery/query.js

```diff
@@ -35,9 +36,20 @@ async function queryZuora (deliveryDate, config: Config) {
      ProductRatePlanCharge.ProductType__c = 'Print ${deliveryDay}' AND
      Product.Name = 'Newspaper Delivery' AND
      RatePlanCharge.EffectiveStartDate <= '${formattedDate}' AND
-     (Subscription.Status = 'Active' AND RatePlanCharge.EffectiveEndDate >= '${formattedDate}' OR
-      Subscription.Status = 'Cancelled' AND RatePlanCharge.EffectiveEndDate > '${formattedDate}'
-     ) AND
+     (
+      ( 
+        Subscription.Status = 'Active' AND Subscription.AutoRenew = true AND RatePlanCharge.EffectiveStartDate >= '${currentDate}' AND RatePlanCharge.EffectiveEndDate >= '${currentDate}'
+      )
+      OR
+      (
+        Subscription.Status = 'Active' AND RatePlanCharge.EffectiveEndDate >= '${formattedDate}'
+      )
+      OR
+      ( 
+        Subscription.Status = 'Cancelled' AND RatePlanCharge.EffectiveEndDate > '${formattedDate}' 
+      )
+     ) 
+     AND
```

Comment:
I don't think we really need to remove the projection conditions for fulfilment with delivery date in the past since:
-  if the sub ended before the delivery date it will be excluded by the condition RatePlanCharge.EffectiveStartDate <= '${formattedDate}'
- if the sub has a rate plan charge that started before or on the delivery date and is still active and has autorenew on then it should be included in the fulfilment file